tests/Backend/SOX/MergeRequestsTest.php

```diff
@@ -1590,9 +1590,12 @@ function ($events) use ($id) {
         );
 
         $devBranch = new DevBranches($this->developerClient);
-        $this->expectException(ClientException::class);
-        $this->expectExceptionMessage(sprintf('Branch id:"%s" was not found.', $id));
-        $this->expectExceptionCode(404);
-        $devBranch->getBranch((int) $id);
+        try {
+            $devBranch->getBranch((int) $id);
+            $this->fail(sprintf('Branch id:"%s" should not exist.', $id));
+        } catch (ClientException $e) {
+            $this->assertSame(404, $e->getCode());
+            $this->assertSame(sprintf('Branch id:"%s" was not found.', $id), $e->getMessage());
+        }
```

Comment:
@jirkasemmler jaký je mezitím rozdíl?

Reply:
no nevedel jsem to, ale on ten phpunit po tom chcipne a nedostane se dal https://stackoverflow.com/a/14563410 . jsem mel nastaveny breakpoint na tom `CANCELED` statu (bo mi to nesedelo a chtel jsem to vyzkouset) a zaboha se to tam nechtelo dostat... tak jsem na to prisel tady

Reply:
a ono to nevadilo dokud se tato metoda pouzila na konci testu. Ale v tomto PR ji pouzivas uprostred

Reply:
To je vlastně pravda, good catch.